src/renderer/components/dock/terminal/terminal-fonts.global-override-for-injectable.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getGlobalOverride } from "../../../../common/test-utils/get-global-override";
+import { preloadAllTerminalFontsInjectable } from "./terminal-fonts.injectable";
+
+export default getGlobalOverride(preloadAllTerminalFontsInjectable, () => {
+  return {
+    id: "",
```

Comment:
nit: might be a good idea to keep the same `id`

Reply:
I intentionally didn't put any id to avoid duplication and keeping in sync with the injectable in future (it feels like it should not be responsible for this, maybe we should not ask `id` at all and use it from passed injectable automatically) cc @jansav

Reply:
This ID is different from the Injectable ID because it is part of the data which is created, same as any other field (could have been call `name`).

Reply:
Why it's different? If I duplicate the string it would be the same. Moreover, we can use different ID from returned object (from runnable):

```ts
export const preloadAllTerminalFontsInjectable = getInjectable({
  id: "preloadAllTerminalFontsInjectable",

  instantiate(di) {
    const terminalFonts = di.inject(terminalFontsInjectable);
    const preloadFont = di.inject(preloadTerminalFontInjectable);

    return {
      id: "preload-all-terminal-fonts", // <--- THAT ONE

      async run() {
        await Promise.allSettled(
          Array.from(terminalFonts.keys()).map(preloadFont),
        );
      },
    };
  },

  injectionToken: beforeFrameStartsFirstInjectionToken,

  causesSideEffects: true,
});
```

Reply:
` id: "preload-all-terminal-fonts", // <--- THAT ONE` This is the one that I am talking about

Reply:
Yes, I mean this ID should be utilized automatically from passed injectable to make `id` optional.

Reply:
Fair enough that would be nicer indeed